.env

```diff
@@ -11,23 +11,27 @@ DEMO_HOST_PREFIX=privacy-sandbox-demos-
 
 # home
 HOME_HOST=privacy-sandbox-demos-home.dev
+HOME_URI=http://privacy-sandbox-demos-home.dev:8080
```

Comment:
On the local docker network, communication between services does not use https & listen to port 8080. This is the reason why we added xxx_URI to include scheme (http:// or https://) & port on top of xxx_HOST. We wanted to know the url of each service for situations where services communicates to each others.
however if you look at GCP env (dev.dev and env.prod) we use https:// port 443 because CloudRun always requires communications through external endpoint.